templates/labelVideo.html

```diff
@@ -20,11 +20,8 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons">
   <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap">
-  <link rel="stylesheet" type="text/css"
-      href="https://storage.googleapis.com/{{project_id}}/static/css/styles.css?ignoreCache={{time_time}}">
-  <script type="text/javascript"
-      src="https://storage.googleapis.com/{{project_id}}/compiled/js/fmltc.js?ignoreCache={{time_time}}">
-  </script>
+  <link rel="stylesheet" type="text/css" href="static/css/styles.css">
+  <script type="text/javascript" src="static/js/fmltc.js"> </script>
```

Comment:
static/js/fmltc.js doesn't exist, does it? Is there a change needed in scripts/deploy_js.sh?

Reply:
To fully support this, there are changes needed to the build.   At present I'm just running the closure compile manually and copying the file over.  I'd work that out before removing the draft label on this pr.